ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-commands.sh

```diff
@@ -157,6 +157,10 @@ if [ ${SECURE_BOOT_ENABLED} = "true" ]; then
   sed -i '/guest_id/a\  firmware         = "efi"\n  efi_secure_boot_enabled = "true"' ./vm/main.tf
 fi
 
+if [ ${VMS_ENCRYPTED} = "true" ]; then
+  sed -i '/guest_id/a\ storage_policy_id = "bff3b0eb-28a9-46f5-baa0-545d02aa3bad"' ./vm/main.tf
```

Comment:
Hmm, we used policy id here, so it seems only adapt the specified vsphere env. What's the policy name of `bff3b0eb-28a9-46f5-baa0-545d02aa3bad`?  Is it called `Management Storage policy - Encryption
`?

Reply:
@Phaow Here it's using hekumar-encrypted policy, if you prefer I can change it to Management Storage policy - Encryption.
sh-4.4$ govc storage.policy.ls | grep -i encrypt
bff3b0eb-28a9-46f5-baa0-545d02aa3bad  hekumar-encrypted
3a8bc349-865f-44c9-be6c-5ff314022c39  SGAO Storage Policy Encrypt
af70d898-2045-4364-a2a0-2fb76e7eb10d  pewang-encrypt-csi
b1263970-8662-69e2-adc6-fa8ae01abecc  Management Storage policy - Encryption

And yes, it must run on IBMC env, I'm trying to appy a new profile to schedule encrypted workflow to IBMC only, details see
https://redhat-internal.slack.com/archives/C015H2WDJRY/p1677222118721439?thread_ts=1677057427.979029&cid=C015H2WDJRY

Reply:
Got it. Thanks for the info! I think we'd better use a unique policy for CI like `qe-ci-encrypted`. I'm not sure if hemant remove the test policy some day.

Reply:
Got it, just got new info from doc review here https://github.com/openshift/openshift-docs/pull/56739/files#r1123608757, based on it let's try with"b1263970-8662-69e2-adc6-fa8ae01abecc Management Storage policy - Encryption"
`yeah, I don't think we should be suggesting folks to create encryption storage policy at this stage. There are bundled encryption policies with vcenter ( https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-431FDB2F-7F34-468D-9D6B-BC5E95279237.html). If people want to use their own encryption storage policies they can, but we don't have to suggest that - at this stage.`

Reply:
Thanks for the info! Then we could use the preset policy `Management Storage policy - Encryption` instead.